llmware/library.py

```diff
@@ -415,7 +558,18 @@ def set_incremental_docs_blocks_images(self, added_docs=0, added_blocks=0, added
     def add_file(self, file_path):
 
         """ Ingests, parses, text chunks and indexes a single selected file to a library -
```

Comment:
Delete space before you start the doc.
```suggestion
        """Ingests, parses, text chunks and indexes a single selected file to a library -
```